src/sentry/models/featureadoption.py

```diff
@@ -0,0 +1,149 @@
+    Feature("pylons", "Pylons", "framework", prerequisite=["python"]),
+    Feature("tornado", "Tornado", "framework", prerequisite=["python"]),
+    Feature("webpy", "web.py", "framework", prerequisite=["python"]),
+    Feature("zope", "Zope", "framework", prerequisite=["python"]),
+
+    # Configuration
+    Feature("first_event", "First Event", "code", prerequisite=["first_project"]),
+    Feature("release_tracking", "Release Tracking", "code", prerequisite=["first_event"]),
+    Feature("environment_tracking", "Environment Tracking", "code", prerequisite=["first_event"]),
+    Feature("user_tracking", "User Tracking", "code", prerequisite=["first_event"]),
+    Feature("custom_tags", "Custom Tags", "code", prerequisite=["first_event"]),
+    Feature("source_maps", "Source Maps", "code", prerequisite=["first_event", "javascript"]),
+    Feature("user_feedback", "User Feedback", "code", prerequisite=["user_tracking"]),
+    Feature("api", "API", "code", prerequisite=["first_event"]),
+    Feature("breadcrumbs", "Breadcrumbs", "code", prerequisite=["first_event", ("python", "javascript", "node", "php")]),
+    # Feature("resolve_in_commit", "Resolve in Commit", "code", prerequisite=["first_event", "releases"]),
+
+    # Web UI
+    Feature("first_project", "First Project", "web"),
+    Feature("invite_team", "Invite Team", "web", prerequisite=["first_project"]),
+    Feature("assignment", "Assign Issue", "web", prerequisite=["invite_team", "first_event"]),
+    Feature("resolved_in_release", "Resolve in Next Release", "web", prerequisite=["release_tracking"]),
+    # Feature("snooze", "Snooze Issue", "web", prerequisite=["first_event"]),
+    # Feature("merge", "Merge Issues", "web", prerequisite=["first_event"]),
+    Feature("advanced_search", "Advanced Search", "web", prerequisite=["first_event"]),
+    Feature("saved_search", "Saved Search", "web", prerequisite=["advanced_search"]),
+    Feature("inbound_filters", "Inbound Filters", "web", prerequisite=["first_event"]),
+    Feature("alert_rules", "Alert Rules", "web", prerequisite=["first_event"]),
+    Feature("issue_tracker_integration", "Issue Tracker Integration", "web", prerequisite=["first_project"]),
+    Feature("notification_integration", "Notification Integration", "web", prerequisite=["first_project"]),
+    # Feature("releases", "Releases", "web", prerequisite=["first_project"]),
+
+    # Admin UI
+    Feature("sso", "SSO", "admin", prerequisite=["invite_team"]),
+    Feature("data_scrubbers", "Data Scrubbers", "admin", prerequisite=["first_event"]),
+
+    # Future features
+    # Feature("Health Metrics", "code"),
+    # Feature("Customer Support Integration", "web", prerequisite=["User Tracking"]),
+    # Feature("ETL Integration", "web", prerequesite=["First Event"]),
+    # Feature("2-Factor Auth", "web", prerequisite=["Invite Team"]),
+]
+
+# No feature slug should be duplicate
+assert len(features) == len(set([f.slug for f in features]))
```

Comment:
If we take my other suggestions and refactor this into a manager pattern, the manager can just handle this automatically like we do elsewhere.